tests/fixture_plugins/demultiplex_fixtures/flow_cell_fixtures.py

```diff
@@ -42,61 +42,32 @@ def hiseq_2500_custom_index_flow_cell(
 
 
 @pytest.fixture()
-def novaseq_6000_post_1_5_kits_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
-    return FlowCellDirectoryData(Path(flow_cells_dir, "230912_A00187_1009_AHK33MDRX3"))
-
-
-@pytest.fixture()
-def novaseq_6000_pre_1_5_kits_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
+def novaseq_6000_pre_1_5_kits_flow_cell(flow_cells_dir: Path) -> FlowCellDirectoryData:
+    """Return a Novaseq6000 flow cell with index settings pre 1.5 kits."""
     return FlowCellDirectoryData(Path(flow_cells_dir, "190927_A00689_0069_BHLYWYDSXX"))
 
 
 @pytest.fixture()
-def novaseq_x_flow_cell_data(flow_cells_dir: Path) -> FlowCellDirectoryData:
-    return FlowCellDirectoryData(Path(flow_cells_dir, "20231108_LH00188_0028_B22F52TLT3"))
-
-
-# Broken flow cells
```

Comment:
Removed broken flow cells from here as I only need the Path objects, not the flow cell objects